app/services/domain/utilisation-date.js

```diff
@@ -0,0 +1,35 @@
+const ValidationError = require('../errors/validation-error')
+const FieldsetValidator = require('../validators/fieldset-validator')
+const ErrorHandler = require('../validators/error-handler')
+const dateFormatter = require('../date-formatter')
+const ERROR_MESSAGES = require('../validators/validation-error-messages')
+
+class UtilisationDate {
+  constructor (day, month, year) {
+    this.fields = [
+      day,
+      month,
+      year
+    ]
+
+    this.utilisationDate = dateFormatter.build(day, month, year)
+    this.IsValid()
+  }
+
+  IsValid () {
+    var errors = ErrorHandler()
+
+    FieldsetValidator(this.fields, 'utilisationDate', errors)
```

Comment:
Okay cool. Can change this based on the change from above. Might need to tag where the data is coming from in this case as so far it just treats it as two of the same type of dates.